### PR TITLE
DX: simplify PHP CS Fixer config towards v3.62

### PR DESCRIPTION
**Pull request by keradus**
| Q             | A
| ------------- | ---
| Branch?       | 7.2
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Issues        | no
| License       | MIT

following #57901

### CODE REVIEW

**Review by OskarStark — APPROVED** [review]

**Comment by derrabus** [comment]
We can also clean up the `native_function_invocation` override, can't we?

**Comment by keradus** [comment]
indeed @derrabus 🙇🏻 
applied to the PR, also don't hesitate to raise such cleanups whenever you are aware of such possibilities

File: .php-cs-fixer.dist.php [hunk 1]
```diff
@@ -32,12 +32,9 @@
         '@Symfony:risky' => true,
         'protected_to_private' => false,
         'header_comment' => ['header' => $fileHeaderComment],
-        // TODO: Remove once the "compiler_optimized" set includes "sprintf"
-        'native_function_invocation' => ['include' => ['@compiler_optimized', 'sprintf'], 'scope' => 'namespaced', 'strict' => true],
         'nullable_type_declaration' => true,
```
Comment (keradus) [inline .php-cs-fixer.dist.php]:
```suggestion
```

File: .php-cs-fixer.dist.php [hunk 2]
```diff
@@ -32,12 +32,9 @@
...
         'nullable_type_declaration_for_default_null_value' => true,
```
Comment (keradus) [inline .php-cs-fixer.dist.php]:
```suggestion
```

**Comment by xabbuh** [comment]
@fabpot Does fabbot need to be updated to use the latest PHP CS fixer version or does that happen automatically?

**Review by derrabus — APPROVED** [review]
👍🏻  as soon as @fabpot has updated fabbot. 🤖

**Comment by OskarStark** [comment]
I think it does not work automatically

**Review by fabpot — APPROVED** [review]

**Review by chalasr — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @keradus.

**Comment by fabpot** [comment]
Thank you @keradus.

**Pull request merged**